pom.xml

```diff
@@ -1776,6 +1776,41 @@
         <findbugs.excludeFilterFile>src/main/findbugs/exclude-filter.xml</findbugs.excludeFilterFile>
       </properties>
     </profile>
+    <profile>
+      <id>overrides</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <compilerId>javac-with-errorprone</compilerId>
+              <forceJavacCompilerUse>true</forceJavacCompilerUse>
+              <source>1.8</source>
+              <target>1.8</target>
+              <compilerArgs>
+                <arg>-XepDisableAllChecks</arg>
+                <arg>-Xep:MissingOverride:ERROR</arg>
+                <arg>-XepExcludedPaths:.*/(proto|thrift|generated-sources)/.*</arg>
```

Comment:
Is this better than `-XepDisableWarningsInGeneratedCode`?

Reply:
Unfortunately, yes. That flag only ignores generated code if certain annotations are in the code. The majority of our generated code does not have the annotation and were not ignored.

Reply:
Can we annotate generated code with the appropriate annotations? Seems like that would be useful for other static analysis tools we use as well.